### ISSUE TITLE
Overflow Menu component accessibility (AVT 3)

### ISSUE THREAD

**Issue by johnharan** [IBM/carbon-components-angular#289]
With JAWS running, tab to the overflow menu button, note JAWS is not reading-out clear/intuitive instructions on how to expand/collapse the overflow menu. JAWS reads out "overflow underlying menu. overflow button collapsed". 

JAWS should say that an actions menu is selected, and how to activate the menu.
E.g. JAWS could read out "Actions menu collapsed. To expand the actions menu, press enter or space".

"Actions Menu" may be a more recognizable description for the user instead of "Overflow".
e.g. "Actions Menu button" instead of "Overflow button".

The AVT3 test case for this issue:
Tab to every control and verify each is spelled out with label and type of control and how to activate the control.

> Is this a feature request (new component, new icon), a bug, or a general issue?

bug

> Is this issue related to a specific component?

Overflow Menu

> What browser are you working in?

Firefox 63.0.3 (64-bit)

> What version of the Carbon Design System are you using?

"carbon-components": "9.53.2",
"carbon-components-angular": "2.0.8",

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

IWR

**Comment by cal-smith** [comment]
Are you still seeing this issue? Testing in JAWS it reads out "Overflow button menu menu press space to activate the menu then navigate with arrow keys" which seems correct to me. 

You definitely should provide a more contextually appropriate aria-label though - we default to "Overflow" but you can get more specific via the `buttonLabel` input or the `OVERFLOW_MENU.OVERFLOW` translation key.

**Comment by johnharan** [comment]
@cal-smith Hi, I wasn't aware of the buttonLabel input, that works well for us. The only other small detail which you may/may-not want to change is to make JAWS say "press **enter or** space bar...", currently it only reads 'space bar', while enter can also open the overflow menu.

**Comment by cal-smith** [comment]
I'm not sure why JAWS doesn't read "press enter or space" but as far as I'm aware that's not something we have direct control over. 

I'll close this now though 👍
